config/http.conf

```diff
@@ -1,18 +1,33 @@
+daemon off;
+
+worker_processes auto;
+worker_shutdown_timeout 240s;
+
+events {
+  multi_accept        on;
```

Comment:
Multi-accept and epoll are the optimal settings for Linux >= 2.6 according to the docs.

Reply:
I saw this but left it out because [of this official post](https://www.nginx.com/blog/performance-tuning-tips-tricks/) ...

> We recommend keeping the default value (off), unless you’re sure there’s a benefit to changing it. Start performance testing with the default value to better measure predictable scale.

There's also [a comment there that where someone says](https://www.nginx.com/blog/performance-tuning-tips-tricks/#comment-5043382241) ...

> There's no simple cut-off point beyond which enabling the accept() mutex brings performance benefits. We have used accept_mutex and multi_accept to tune NGINX to give more stable, predictable performance under synthetic benchmarks, but this does not typically apply to production workloads. Some users have applied it in production.
>
> Your mileage will vary. If you're seeing varying latency or high CPU usage on NGINX under high traffic load (new connections), and you cannot scale your hardware, enabling it might make a difference. Leave it disabled unless you measure a clear performance difference, and accept that it rarely brings a clear benefit.

I see that it's on by default in Ingress Nginx Kubernetes and theres' not discussion around why (checked https://github.com/kubernetes/ingress-nginx/issues/2684 and https://github.com/kubernetes/ingress-nginx/pull/2896).

Reply:
Hmm, I see. We'd need to run some tests to determine the impact, but I thought that since ingress-nginx has it enabled it can't be too bad.

What do you want to do?

Reply:
> Hmm, I see. We'd need to run some tests to determine the impact, but I thought that since ingress-nginx has it enabled it can't be too bad.

Yes. I agree with you here.

> What do you want to do?

We can leave it as is. No objections from me.